### PR TITLE
fix(circleci): update ci

### PR DESCRIPTION
**Pull request by WikiRik**
Comparable to the other modules, based on the work already deployed on statutory

### CODE REVIEW

File: .husky/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+_
```
Comment (serge1peshcoff) [inline .husky/.gitignore:1]: can you add newlines everywhere?

Comment (WikiRik) [reply .husky/.gitignore]: Fixed, thanks!

Comment (serge1peshcoff) [inline .husky/.gitignore:1]: this should be just empty line and not the underscore, right?

Comment (WikiRik) [reply .husky/.gitignore:1]: No, hubot puts a file in that folder and that needs to be ignored by git. This gitignore is automatically generated by husky

**Review by serge1peshcoff — APPROVED** [review]

**Pull request merged**